docs/index.mdx

```diff
@@ -15,9 +15,17 @@ Storybook is a standalone tool that runs alongside your app. It's a zero-config
 
 <CodeSnippets path="create-command.md" />
 
+Want to know more about installing Storybook? Check out the [installation guide](./get-started/install.mdx).
+
+## Supported frameworks
+
 <HomeRenderers />
 
-Want to know more about installing Storybook? Check out the [installation guide](./get-started/install.mdx).
+## Community-maintained frameworks
+
+External contributors maintain Storybook for additional frameworks. Storybook does not directly provide support for these.
```

Comment:
@Sidnioulz, for the community frameworks, I was thinking of something more along the lines of the following:
```suggestion
Storybook includes an active community that supports additional frameworks and libraries. These community-maintained frameworks are actively developed and maintained by community contributors.
```

Reply:
Thanks, that's much better